From bafa84b25a265ef9eed3872790d52bf56ac42998 Mon Sep 17 00:00:00 2001
From: Arvin Schnell <aschnell@suse.com>
Date: Wed, 27 Jul 2022 09:36:54 +0000
Subject: [PATCH 06/13] libparted: Fix check for availability of _type_id
 functions

Fix a copy/paste error. In practice this didn't cause any problems
because the *_set_type_id and *_get_type_id are either both NULL or both
set to the function.

Signed-off-by: Brian C. Lane <bcl@redhat.com>
---
 libparted/disk.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libparted/disk.c b/libparted/disk.c
index 22dff36..a961d65 100644
--- a/libparted/disk.c
+++ b/libparted/disk.c
@@ -1572,7 +1572,7 @@ ped_partition_get_type_id (const PedPartition *part)
         if (!_assert_partition_type_id_feature (part->disk->type))
                 return 0;
 
-        PED_ASSERT (part->disk->type->ops->partition_set_type_id != NULL);
+        PED_ASSERT (part->disk->type->ops->partition_get_type_id != NULL);
         return part->disk->type->ops->partition_get_type_id (part);
 }
 
@@ -1608,7 +1608,7 @@ ped_partition_get_type_uuid (const PedPartition *part)
         if (!_assert_partition_type_uuid_feature (part->disk->type))
                 return NULL;
 
-        PED_ASSERT (part->disk->type->ops->partition_set_type_uuid != NULL);
+        PED_ASSERT (part->disk->type->ops->partition_get_type_uuid != NULL);
         return part->disk->type->ops->partition_get_type_uuid (part);
 }
 
-- 
2.37.3

